screenpy_selenium/__init__.py

```diff
@@ -29,9 +29,9 @@
 __all__ = [
     "BrowsingError",
     "Chainable",
-    "settings",
     "Target",
     "TargetingError",
+    "settings",
```

Comment:
ruff wants these in slightly different alphabetical order.  Not sure we can do anything to alter that.

Reply:
Makes sense, since `settings` is lower-case.